src/plugins/intel_cpu/tests/functional/subgraph_tests/src/reshape_chain.cpp

```diff
@@ -19,28 +18,29 @@ class ReshapeChain : public SubgraphBaseTest {
         InputShape inputShapes{{-1, -1, -1, -1}, {{10, 20, 30, 40}, {16, 24, 16, 24}, {4, 8, 12, 16}}};
 
         init_input_shapes({inputShapes});
-        auto ngPrc = ngraph::element::f32;
-        const auto secondInPrc = ngraph::element::Type_t::i32;
+        auto ngPrc = ov::element::f32;
+        const auto secondInPrc = ov::element::Type_t::i32;
         ov::ParameterVector inputParams;
         for (auto&& shape : inputDynamicShapes) {
             inputParams.push_back(std::make_shared<ov::op::v0::Parameter>(ngPrc, shape));
         }
         auto reshapeParam1 = ngraph::builder::makeConstant<int>(secondInPrc, {3}, {0, 0, -1});
-        auto reshape1 = std::make_shared<ngraph::opset1::Reshape>(inputParams.front(), reshapeParam1, true);
+        auto reshape1 = std::make_shared<ov::op::v1::Reshape>(inputParams.front(), reshapeParam1, true);
```

Comment:
```suggestion
    auto reshape1 = std::make_shared<ov::opset1::Reshape>(inputParams.front(), reshapeParam1, true);
```

Reply:
> 

@riverlijunjie  Why have to keep "opset*"? what is the benefit?

Reply:
Let's use versions instead of opsets where possible